WcaOnRails/app/javascript/edit-schedule/EditVenue/index.jsx

```diff
@@ -37,9 +37,12 @@ export class EditVenue extends React.Component {
     let newLat = toMicrodegrees(pos.lat);
     let newLng = toMicrodegrees(pos.lng);
     // Update parent's WCIF
-    this.props.venueWcif.latitudeMicrodegrees = newLat;
-    this.props.venueWcif.longitudeMicrodegrees = newLng;
-    rootRender();
+    if (this.props.venueWcif.latitudeMicrodegrees !== newLat
```

Comment:
Out of curiosity, was unconditionally calling `rootRender` causing an infinite re-render loop or something?

Reply:
Yeah, my guess is that rendering the map and marker somehow triggered marker `onMove`. This was an easy fix and makes sense anyway ^^